syncmaster/backend/middlewares/session.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+import secrets
+
+from fastapi import FastAPI
+from starlette.middleware.sessions import SessionMiddleware
+
+
+def apply_session_middleware(app: FastAPI) -> FastAPI:
+    """Add SessionMiddleware middleware to the application."""
+    secret_key = secrets.token_urlsafe(32)
```

Comment:
This should be read from server settings. Otherwise each time server is restarted, all the cookies are reset

Reply:
Other SessionMiddleware settings probably should be configurable too: https://www.starlette.io/middleware/#sessionmiddleware